Org View: Change nav

MarcogodoyV opened the pull request:
Made the changes to de nav, adding the CirclesPage to make the Circles Nav item work, redundant items removed from the Avatar menu (rigth now added to the CirclesPage in order to have a way to switch circles, knowing that after everything is done it's going to work different), I also changed the mobile view. This PR can fix #555

levity commented:
hi @MarcogodoyV, thanks for the PR. can you please merge changes from the base `org-view` branch back into your branch? we recently made a change to the Vercel deployment process that requires a change from there.

levity commented:
> hi @MarcogodoyV, thanks for the PR. can you please merge changes from the base `org-view` branch back into your branch? we recently made a change to the Vercel deployment process that requires a change from there.

Never mind, looks like I was able to do it :)

levity commented on src/routes/paths.ts:
```diff
@@ -143,12 +149,24 @@ export const getMainNavigation = ({
   return mainItems;
 };
 
+export const getAdminNavigation = (): INavItem[] => [
```
I know you're probably just following the existing pattern, but these data structures are small & simple enough that they can just be embedded directly in the components that use them.

levity replied:
@MarcogodoyV this^ is the last thing to clean up, & then it looks good to merge

levity commented on src/routes/paths.ts:
```diff
@@ -45,6 +45,7 @@ export const paths = {
   vaultTxs: (id: string) => `${paths.vaults}/${id}/txs`,
   adminCircles: '/admin/circles',
   connectIntegration: '/connect-integration',
+  circles: '/circles',
```
feel free to start using the paths here instead of the getter methods

levity reviewed (changes requested):
Looks pretty good so far! Just a few requests below. The only thing that seemed off visually was the vertical alignment of the breadcrumb:

![Coordinape](https://user-images.githubusercontent.com/193667/157996655-905cce44-69d5-45da-bfa8-0a27fd9336fa.png)

The content of the CirclesPage is unstyled too, but I assume you're well aware of that and it'll be cleaned up in a future PR.

By the way, we've been switching over to Stitches for styling on new feature development. See [this comment](https://github.com/coordinape/coordinape/pull/584#issuecomment-1061269158) for more context. If you're just making minor tweaks to existing components, feel free to keep using Material-UI, but any more significant changes should be done with Stitches going forward. My experience with Stitches so far is that it's more pleasant to use; I'd love to hear your feedback on it if you try it out too.

MarcogodoyV commented:
@levity thanks for the review!, I did 2 new commits  that I hope gets all done, and also added the breadcrumb in the mobile view. About the CirclesPage I have that issue assign so it's going to be make all new in a new PR ( I added the page in order to have a temporaly way to move across circles)

levity reviewed (approved):
Looks good. Make sure to remove the unused `get*Navigation` methods in routes/paths soon in later PRs
